src/overlays/kaleido_scope/ovl_kaleido_scope/z_kaleido_debug.c

```diff
@@ -1,25 +1,1002 @@
+        }
+        sCurSection = sRowFirstSections[sCurRow];
+    } else if (CHECK_BTN_ANY(dBtnInput, BTN_DLEFT)) {
+        sCurSection--;
+        if (sCurSection < 0) {
+            sCurSection = 92;
+        }
+    } else if (CHECK_BTN_ANY(dBtnInput, BTN_DRIGHT)) {
+        sCurSection++;
+        if (sCurSection > 92) {
+            sCurSection = 0;
+        }
+    }
+
+    // Handles the logic to change values based on the selected section
+    switch (sCurSection) {
+        case 0:
+            // Rupees
+            if (CHECK_BTN_ALL(input->press.button, BTN_CUP)) {
+                gSaveContext.save.playerData.rupees -= 100;
+                if (gSaveContext.save.playerData.rupees < 0) {
+                    gSaveContext.save.playerData.rupees = 0;
+                }
+            } else if (CHECK_BTN_ALL(input->press.button, BTN_CDOWN)) {
+                gSaveContext.save.playerData.rupees += 100;
+                if (gSaveContext.save.playerData.rupees >= 9999) {
+                    gSaveContext.save.playerData.rupees = 9999;
+                }
+            } else if (CHECK_BTN_ALL(input->press.button, BTN_CLEFT)) {
+                gSaveContext.save.playerData.rupees--;
+                if (gSaveContext.save.playerData.rupees < 0) {
+                    gSaveContext.save.playerData.rupees = 0;
+                }
+            } else if (CHECK_BTN_ALL(input->press.button, BTN_CRIGHT)) {
+                gSaveContext.save.playerData.rupees++;
+                if (gSaveContext.save.playerData.rupees >= 9999) {
+                    gSaveContext.save.playerData.rupees = 9999;
+                }
+            }
+            break;
+
+        case 1:
+            // Health capacity
+            if (CHECK_BTN_ALL(input->press.button, BTN_CUP) || CHECK_BTN_ALL(input->press.button, BTN_CLEFT)) {
+                gSaveContext.save.playerData.healthCapacity -= 0x10;
+                if (gSaveContext.save.playerData.healthCapacity < 0x30) {
+                    gSaveContext.save.playerData.healthCapacity = 0x30;
+                }
+            } else if (CHECK_BTN_ALL(input->press.button, BTN_CDOWN) ||
+                       CHECK_BTN_ALL(input->press.button, BTN_CRIGHT)) {
+                gSaveContext.save.playerData.healthCapacity += 0x10;
+                if (gSaveContext.save.playerData.healthCapacity >= 0x140) {
+                    gSaveContext.save.playerData.healthCapacity = 0x140;
+                }
+            }
+            break;
+
+        case 2:
+            // Health
+            if (CHECK_BTN_ALL(input->press.button, BTN_CLEFT)) {
+                Health_ChangeBy(play, -4);
+            } else if (CHECK_BTN_ALL(input->press.button, BTN_CRIGHT)) {
+                Health_ChangeBy(play, 4);
+            } else if (CHECK_BTN_ALL(input->press.button, BTN_CUP)) {
+                Health_ChangeBy(play, -0x10);
+            } else if (CHECK_BTN_ALL(input->press.button, BTN_CDOWN)) {
+                Health_ChangeBy(play, 0x10);
```

Comment:
```suggestion
                Health_ChangeBy(play, -16);
            } else if (CHECK_BTN_ALL(input->press.button, BTN_CDOWN)) {
                Health_ChangeBy(play, 16);
```

Reply:
These should remain hex because heath is much easier to read in hex (`0x10` is 1 heart, `0x30` is 3 hearts, `0xA0` is 10 hearts, etc...).